documentation/src/docs/recovery-procedures/metadata-table-segment-recovery.md

```diff
@@ -0,0 +1,120 @@
+limitations under the License.
+-->
+
+
+# Metadata Table Segment Attribute Index Recovery
+
+
+The below outlined steps refer to recovery of Attribute Index Table Segments but only in cases of Metadata Segments which are a special case, as they need to
+be handled a little differently. Metadata Segments in Pravega are Table Segments, that hold metadata about all other "non-metadata" Segments, which
+include the internal Segments as well as the User Segments. Metadata about these Metadata Segments themselves in Pravega, are stored in Pravega Journals. More details
+about the why and how part of Journals can be found in this PDP [here](https://github.com/pravega/pravega/wiki/PDP-34-(Simplified-Tier-2)#why-slts-needs-system-journal).
+Pravega Journals, is concretely what we would be attempting to update as part of the recovery procedure in this document.
+
+
+Please note that the steps outlined below are in continuation to the ones outlined as part of the recovery procedure described [here](https://github.com/pravega/pravega/blob/master/documentation/src/docs/recovery-procedures/table-segment-recovery.md).
+That is, we need to perform some of the steps mentioned in the above referenced doc, before continuing any of the steps present here.
+
+
+It is precisely at this point [here](https://github.com/pravega/pravega/blob/master/documentation/src/docs/recovery-procedures/table-segment-recovery.md#important) in 
+the Table Segment Recovery procedure, that one would have to jump to the below steps in the case of dealing with Metadata Segments.
+
+
+# Detailed Steps
+1) Once we determine that the Table Segment under repair is a Metadata Table Segment, we copy over the Attribute Index Chunks generated 
+   in the output directory of [Step 4](https://github.com/pravega/pravega/blob/a5088a464275d5ea90adb09ac39027332e87a8e3/documentation/src/docs/recovery-procedures/table-segment-recovery.md?plain=1#L129) to a separate directory of our choice.
+
+
+2) Open the Admin CLI, and disable Tier-1 for the Container owning the Metadata Segment. The owning Container ID can be figured out from the Metadata Segment Chunk 
+   name. For example, if one of the Metadata Chunk file is say `metadata_2.E-1-O-0.88990c40-58a3-463e-8936-652d89fa95ba` then the container is 2 from `metadata_2` in file name.
+   To disable the Tier-1, run the below command:
+   ```
+       bk disable <containerId>
+       Ex:-
+          bk disable 2
+   ```
+
+
+3) Identify the latest Journal file for the affected Container. One can identify the latest Journal file by simply listing the Journal files for the affected Container. 
+   Before listing the Journal files, go to the directory `/mnt/tier2/_system/containers` where `/mnt/tier2` is the configured Tier2 directory 
+   as mentioned in [Step2](https://github.com/pravega/pravega/blob/a5088a464275d5ea90adb09ac39027332e87a8e3/documentation/src/docs/recovery-procedures/table-segment-recovery.md?plain=1#L70).
```

Comment:
The link is also showing plain test, not the rendered file.

Reply:
as i am linking to specific locations within the doc, the view just lands automatically in "source blob" mode( and not the "rendered blob" mode).